holoviews/core/spaces.py

```diff
@@ -594,16 +594,15 @@ def _validate_key(self, key):
 
     def event(self, trigger=True, **kwargs):
         """
-        This method allows any of the available stream parameters to be
-        updated in an event.
+        This method allows any of the available stream parameters
+        (renamed as appropriate) to be updated in an event.
         """
         stream_params = set(util.stream_parameters(self.streams))
         updated_streams = []
         for stream in self.streams:
-            overlap = set(stream.params().keys()) & stream_params & set(kwargs.keys())
-            if overlap:
-                stream.update(**dict({k:kwargs[k] for k in overlap}, trigger=False))
-                updated_streams.append(stream)
+            rkwargs = util.rename_stream_kwargs(stream, kwargs, reverse=True)
+            stream.update(**dict(rkwargs, trigger=False))
```

Comment:
Maybe I'm not following the logic correctly in the utilities but won't this try to send the supplied kwargs to all the streams even though they may not apply? That's what the overlap bit was for I assume.